test/python/circuit/library/test_functional_pauli_rotations.py

```diff
@@ -17,11 +17,13 @@
 import numpy as np
 from ddt import ddt, data, unpack
 
+from qiskit import transpile
 from qiskit.circuit import QuantumCircuit
 from qiskit.circuit.library import (
     LinearPauliRotations,
     PolynomialPauliRotations,
     PiecewiseLinearPauliRotations,
+    PiecewiseLinearPauliRotationsGate,
```

Comment:
should there be a test for LinearPauliRotationsGate ? PolynomialPauliRotations ?

Reply:
Yeah that'd be good, we can extend the tests for the `PiecewiseLinearPauliRotationGate` to also check others.